### PR TITLE
Logic file tempfix

### PR DESCRIPTION
**Pull request by Carly-Klebine**
Fixed code so that InputManager works as intended

### CODE REVIEW

File: src/PlayerActor.cpp [hunk 1]
```diff
@@ -5,22 +5,22 @@ PlayerActor::PlayerActor(){
 }
 
 void PlayerActor::init(){
-  position = Position();
-  velocity = sf::Vector2f(50.f,50.f);
+  position = Position(400.f, 300.f);
+  myVelocity = sf::Vector2f(8000.f,8000.f);
```
Comment (alexchung1233) [inline src/PlayerActor.cpp:9]: change to playerVelocity

File: src/GameView.cpp [hunk 2]
```diff
@@ -12,25 +12,61 @@ using namespace std;
...
-  inputManager.update(Event);
-  sprite.setPosition(logic->getPlayer().getPosition().x,
-   logic->getPlayer().getPosition().y);
-  App->draw(sprite);
+
+
+void GameView::update(sf::Event& Event, float dt){
+  //this->App->clear();
+  inputManager.update(Event, dt);
+
```
Comment (alexchung1233) [inline src/GameView.cpp:56]: have player = inputManager.logic.getPlayer() here

File: src/Game.cpp [hunk 3]
```diff
@@ -40,6 +40,8 @@ void Game::render(){
 void Game::initGameView(){
   this->gameView = GameView(*window);
   this->gameView.setup();
+  this->gameView.myPos();
```
Comment (alexchung1233) [inline src/Game.cpp:43]: we can probably remove this here

**Review by JackAkers — APPROVED** [review]

File: src/GameView.cpp [hunk 4]
```diff
@@ -12,25 +12,61 @@ using namespace std;
...
-void GameView::update(sf::Event Event){
-  App->clear();
-  inputManager.update(Event);
-  sprite.setPosition(logic->getPlayer().getPosition().x,
-   logic->getPlayer().getPosition().y);
-  App->draw(sprite);
+
+
+void GameView::update(sf::Event& Event, float dt){
+  //this->App->clear();
```
Comment (alexchung1233) [inline src/GameView.cpp:54]: this should be uncommented

**Review by alexchung1233 — CHANGES_REQUESTED** [review]

File: src/GameView.cpp [hunk 5]
```diff
@@ -12,25 +12,61 @@ using namespace std;
...
+  string player_file = "../data/protag_V1.png";
+
+  if(!texture_player.loadFromFile(player_file)){
+    printf("incorrect file format");
+  }
+  
   sprite.setTexture(texture);
+  sprite_player.setTexture(texture_player);
+
+  sprite_player.setPosition(inputManager.logic.getPlayer().getPosition().x, inputManager.logic.getPlayer().getPosition().y);
```
Comment (alexchung1233) [inline src/GameView.cpp:44]: and use player instead of getPlayer() here

**Review by ktkel — APPROVED** [review]

**Pull request merged**